gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/model/Gaffer.java

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-package uk.gov.gchq.gaffer.common.model.v1;
```

Comment:
Why is v1 being removed?

Reply:
Sorry error while refactoring. Resolved.